### DISCUSSION TITLE
Error in Pascal Voc Evaluator (classes_frames_gt_bbs)

### DISCUSSION THREAD

**Discussion by andrearakel** [rafaelpadilla/review_object_detection_metrics#95] [Q&A]
<img width="547" alt="errormsg" src="https://user-images.githubusercontent.com/12718379/157031703-32bd0050-784d-4f16-9ab6-fc4e73c2b349.png">

I get this error message after clicking 'run'. Both show ground-truth statistics and show detection statistics output correct bbox and detections. Any idea what can be the issue here?

**Comment by rafaelpadilla** [comment]
Hi,

The problem might be related to the names of your images and files. Besides the "." that separates filename and extension (e.g. "my_image.jpg"), they seem to contain "." as in "my.image.jpg".

Please, try to remove the "." of your files names and let me know if that solves your problem.
